src/Components/LpBackground.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import './LpBackground.css'
+
+function LpBackground() {
+  return (
+      <div className='background'>
+    </div>
+  )
+}
```

Comment:
This can be a self-closing div
```suggestion
  return <div className='background' />;
}
```